docs/installation/sync-setup.md

```diff
@@ -8,24 +8,33 @@
 
 The synchronization is case-sensitive so make sure you don't mess up with the capitalization
 
-If you use LuckPerms, these will be the group display names
-
 ```yaml
 # Example from generic permissions plugin
 groups:
-  MinecraftGroup1:
+  VIP:
+    permissions
+    ...
+  MVP:
+    permissions
+    ...
+  Admin:
     permissions
     ...
-  MinecraftGroup2:
+  Owner:
     permissions
     ...
 ```
 
+If you are using LuckPerms, it will look similar to this:
+
+![luckperms groups](/assets/luckperms_groups.png)
+
+
```

Comment:
Perhaps make this a drop down with a title like "Using LuckPerms for groups?" and then specify that the display name is used, stating that its the one in brackets.